ui/src/selectors/annotation.ts

```diff
@@ -23,10 +25,10 @@ export function divDimensionStyles(pageView: PDFPageView, box: BoundingBox) {
    * https://github.com/wojtekmaj/react-pdf/blob/73f505eca1bf1ae243a2b7068fce1e86b98b408a/src/Page/AnnotationLayer.jsx#L104
    */
   return {
-    left: viewportBox[0],
+    left: viewportBox[0] - WIDTH_CORRECTION,
     top: viewportBox[1],
-    width: viewportBox[2] - viewportBox[0],
-    height: viewportBox[1] - viewportBox[3]
+    width: viewportBox[2] - viewportBox[0] + (2 * WIDTH_CORRECTION),
+    height: viewportBox[1] - viewportBox[3] + HEIGHT_CORRECTION
```

Comment:
Can we do these height and width corrections using margin and padding, instead of dynamically setting width and height here? That way the hit-boxes can be swapped out dynamically. Hit boxes sizing is currently implemented here, with the `@highlight-hitbox-padding` parameter: https://github.com/allenai/scholar-reader/blob/23a1fd4c3748e08c7e95ca694595ff437cd36181/ui/src/style/annotation.less#L11-L14